scheduler/src/main/java/org/cloudfoundry/autoscaler/scheduler/service/ScheduleManager.java

```diff
@@ -408,13 +429,28 @@ private SpecificDateScheduleDateTime validateStartEndDateTime(String scheduleBei
 	public void createSchedules(ApplicationScalingSchedules applicationScalingSchedules) {
 
 		List<SpecificDateScheduleEntity> specificDateSchedules = applicationScalingSchedules.getSpecific_date();
-		for (SpecificDateScheduleEntity specificDateScheduleEntity : specificDateSchedules) {
-			// Persist the schedule in database
-			SpecificDateScheduleEntity savedScheduleEntity = saveNewSpecificDateSchedule(specificDateScheduleEntity);
+		if (specificDateSchedules != null) {
+			for (SpecificDateScheduleEntity specificDateScheduleEntity : specificDateSchedules) {
+				// Persist the schedule in database
+				SpecificDateScheduleEntity savedScheduleEntity = saveNewSpecificDateSchedule(
+						specificDateScheduleEntity);
 
-			// Ask ScalingJobManager to create scaling job
-			if (savedScheduleEntity != null) {
-				scalingJobManager.createSimpleJob(savedScheduleEntity);
+				// Ask ScalingJobManager to create scaling job
+				if (savedScheduleEntity != null) {
+					scalingJobManager.createSimpleJob(savedScheduleEntity);
+				}
+			}
+		}
+
+		List<RecurringScheduleEntity> recurringScheduls = applicationScalingSchedules.getRecurring_schedule();
+		if (recurringScheduls != null) {
+			for (RecurringScheduleEntity recurringScheduleEntity : recurringScheduls) {
+				// Persist the schedule in database
+				RecurringScheduleEntity savedScheduleEntity = saveNewRecurringSchedule(recurringScheduleEntity);
+
+				// Ask ScalingJobManager to create scaling job
+				if (savedScheduleEntity != null) {
```

Comment:
Be supposed to implement to create cron job with Quartz on the story 'When recurring rules are created, associated jobs are created'.
Will remove the 'if' statement.